common.yml

```diff
@@ -532,17 +532,6 @@ functions:
 
           echo -n "$MONGOD_URI" > MONGOD_URI
 
-  "check third-party notices":
-    - command: shell.exec
-      type: test
-      params:
-        working_dir: src/github.com/mongodb/mongo-tools
-        script: |
-          set -e
-          set -x
-          ./etc/generate-notices.pl > ./THIRD-PARTY-NOTICES
-          git diff --exit-code
```

Comment:
Before this checked that the notices were correct, and now it only writes them; is that what we want? (It seems like not.)

Reply:
Oops, good catch! Yes, it should check for a diff. Fixed.